src/main/resources/application.yml

```
@@ -0,0 +1,36 @@
+  servlet:
+    multipart:
+      enabled: true
+      max-request-size: 5MB
+      max-file-size: 5MB
+
+jwt:
+  access:
+    expiry-millis: 3600000
+  refresh:
+    expiry-millis: 40000000
+  secret: rptKgP2Ra47xx2yV2OPRYJpuf8QJzsrzOPjxuXs3E5IMOoP5BE1zFSkZA+Th1O6F8XGsU+hwoyn4HAVNe9QoTQ==
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)